cli/src/main.rs

```diff
@@ -81,6 +92,17 @@ helm uninstall linkerd-failover"
                 std::process::exit(1);
             }
         }
+        Commands::Status {
+            ref output,
+            label_selector,
+        } => {
+            let client = cli.client.try_client().await?;
+            let results = status::status(client, label_selector.as_str()).await?;
+            match output {
+                OutputMode::Table => status::print_status(results),
+                OutputMode::Json => status::json_print_status(results),
```

Comment:
And then this would need to borrow

```suggestion
                OutputMode::Table => status::print_status(&results),
                OutputMode::Json => status::json_print_status(&results),
```